docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,382 @@
+
+.. code-block::
+
+    sudo apt-get install python3-venv
+    # create virtual env
+    bash ./otx/algorithms/detection/init_venv.sh det_venv
+    # activate virtual env
+    source det_venv/bin/activate
+
+.. #TODO: Update installation part: virtual env, install prerequisite, pip install -e . or pip install otx
+
+***************************
+Dataset preparation
+***************************
+
+1. Clone a repository with `WGISD dataset <https://github.com/thsant/wgisd>`_. 
+
+.. code-block::
+
+  cd data
+  git clone https://github.com/thsant/wgisd.git
+  cd wgisd
+  git checkout 6910edc5ae3aae8c20062941b1641821f0c30127
+
+
+This dataset contains images of grapevines with the annotation for different varieties of grapes. 
+
+- ``CDY``	- Chardonnay
+- ``CFR``	- Cabernet Franc
+- ``CSV``	- Cabernet Sauvignon
+- ``SVB``	- Sauvignon Blanc
+- ``SYH``	- Syrah
+
+It's a great example to start with. The model achieves high accuracy righ from the beginning of the training due to relatively large and focused objects. Also, these objects are distinguished by a person, so we can check inference results just by looking at images.
+
+.. image:: ../../../../../utils/images/wgisd_dataset_sample.jpg
+  :width: 600
+  :alt: this image uploaded from this `source <https://github.com/thsant/wgisd/blob/master/data/CDY_2015.jpg>`_
+
+
+2. Check the file structure of downloaded repository, we will need the following files:
+
+.. code-block::
+
+  wgisd
+  ├── README.md
+  ├── coco_annotations/
+      ├── train_bbox_instances.json
+      └── test_bbox_instances.json
+  ├──data/
+      └── <images>
+  ...
+
+
+3. ``(Optional)`` To simplify the command line functions calling, we may create a ``data.yaml`` file with annotations info and pass it as a ``--data`` parameter. The content of the ``training_extesions/data.yaml`` for WGISD dataset should have absolute paths and will be similar to that:
+
+.. code-block::
+
+  {'data': 
+    {'train': 
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/train_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'},
+    'val':
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/test_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'},
+    'test':
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/train_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'}
+    }
+  }
+
+``Ann-files`` contains a path to the annotation, while ``data-roots`` is a path to the folder, where images are stored.
+
+*********
+Training
+*********
+
+1. First of all, we need to choose which object detection model will we train. The list of supported templates for object detection is available with the command line below. 
```

Comment:
What do you think about the changing the indentation level?
![image](https://user-images.githubusercontent.com/80735344/213086002-8beba659-c5f6-4d44-a1c7-6ab0af312ee7.png)

Reply:
Good point with indents, but it's again the way how GitHub renders this. After building it looks much better :) 
![image](https://user-images.githubusercontent.com/33455576/213126338-8c8fc163-bf56-4c73-a342-16b853ba27f2.png)